cs.py

```diff
@@ -220,7 +220,10 @@ def main():
         key, value = option.split('=', 1)
         kwargs[key].add(value.strip(" \"'"))
 
-    region = args.get('region', 'cloudstack')
+    if 'CLOUDSTACK_REGION' in os.environ:
+        region = args.get('region', os.environ['CLOUDSTACK_REGION'])
+    else:
+        region = args.get('region', 'cloudstack')
```

Comment:
Without the if/else:

`region = args.get('region', os.environ.get('CLOUDSTACK_REGION', 'cloudstack'))`